@here/harp-mapview/lib/MapView.ts

```diff
@@ -641,6 +670,8 @@ export const MapViewDefaults = {
  * linked to datasources.
  */
 export class MapView extends THREE.EventDispatcher {
+    static defaultUriResolver?: UriResolver;
```

Comment:
Why is this one optional? Shouldn't there always be a default one?

Reply:
This was not necessary, removed completly.